modules/core/src/main/java/org/apache/ignite/configuration/ThinClientConfiguration.java

```diff
@@ -103,21 +103,42 @@ public ThinClientConfiguration setMaxActiveComputeTasksPerConnection(int maxActi
 
     /**
      * @return If {@code true} sends a server exception stack to the client side.
+     * @deprecated Use {@link #getServerExceptionStackTraceToClient()} instead.
      */
+    @Deprecated
     public boolean sendServerExceptionStackTraceToClient() {
         return sendServerExcStackTraceToClient;
     }
 
     /**
      * @param sendServerExcStackTraceToClient If {@code true} sends a server exception stack to the client side.
      * @return {@code this} for chaining.
+     * @deprecated Use {@link #setServerToClientExceptionStackTraceSending(boolean)} instead.
      */
+    @Deprecated
     public ThinClientConfiguration sendServerExceptionStackTraceToClient(boolean sendServerExcStackTraceToClient) {
         this.sendServerExcStackTraceToClient = sendServerExcStackTraceToClient;
 
         return this;
     }
 
+    /**
+     * @return If {@code true} gets a server exception stack that is to be sent to the client side.
+     */
+    public boolean getServerExceptionStackTraceToClient() {
```

Comment:
The name should be the same as for setter. Javadoc also must be fixed. 
BTW do we really need getter with `get` prefix? We need setter to set value in spring XML configuration, but getter is not used here, pehaps the new getter is redundant?

Reply:
Thank you, I will rename the method to be the same as for setter and correct Javadoc